### PR TITLE
DDF-4275 Fixed authz config in itests

### PR DESCRIPTION
**Pull request by bdeining**
#### What does this PR do?
Fixed authz config in itests
#### Who is reviewing it? 

@Bdthomson @Schachte 
#### Select relevant component teams: 
@codice/build 
#### Ask 2 committers to review/merge the PR and tag them here.

@clockard
@figliold
#### How should this be tested?
Full Build

#### What are the relevant tickets?
[](https://codice.atlassian.net/browse/)
#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by bdeining** [comment]
build now

File: distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/config/ConfigureTestCommons.java [hunk 1]
```diff
@@ -100,7 +100,7 @@ public static void configureEnforcedMetacardValidators(
   public static Dictionary<String, Object> configureAuthZRealm(
       List<String> attributes, AdminConfig configAdmin) throws IOException {
     Dictionary<String, Object> properties = new DictionaryMap<>();
```
Comment (figliold) [inline distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/config/ConfigureTestCommons.java]: ❓ Should we be including all of the other properties.  For example, matchAllMappings, etc.?

Comment (bdeining) [reply distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/config/ConfigureTestCommons.java]: There currently are no itests that use those optional configuration properties

Comment (figliold) [reply distribution/test/itests/test-itests-common/src/main/java/org/codice/ddf/itests/common/config/ConfigureTestCommons.java]: This fixes the current issue, but I guess at some point we should update CoinfigureTestCommons.configureAuthZRealm() to configure all of the properties.

**Comment by GabrielFabian** [comment]
build now

**Comment by bdeining** [comment]
closing in favor of https://github.com/codice/ddf/pull/4010

**Pull request closed without merging**